src/pages-and-resources/discussions/app-list/FeaturesList.jsx

```diff
@@ -34,10 +32,9 @@ function FeaturesList({ app, features, intl }) {
     >
       {features && features.map((feature) => (
         <div key={`collapsible-${app.id}&${feature.id}`} className="d-flex mb-1">
-          {app.featureIds.includes(feature.id)
-            ? SupportedFeature
-            : NonSupportedFeature}
-          {intl.formatMessage(messages[`featureName-${feature.id}`])}
+          {app.featureIds.includes(feature.id) && (
+            SupportedFeature(intl.formatMessage(messages[`featureName-${feature.id}`]))
```

Comment:
seems like `SupportedFeature` is component, so you should call it as 
```
<SupportedFeature name="foo" />
```

also as you are in a loop, make sure you add a `key` attrib as well.

Reply:
if its a function it should have a camel case name. 
```
const renderSupportedFeature =  (name) => ()
```
If you make it a component, the code will be much more readable and self intuitive.